chisel/src/cmd_oneliner.rs

```diff
@@ -0,0 +1,119 @@
+//! Unix-style chisel mode implementation.
+//! The main entry point is chisel_oneliner, which uses FromArgs to produce ChiselConfig
+//! from the relevant options passed in the CLI.
+//! Like config-driven mode, it then passes the config to the driver, executes, and writes
+//! output to the specified file (or stdout, if no file is specified).
+
+use crate::config::ChiselConfig;
+use crate::config::FromArgs;
+use crate::driver::{ChiselDriver, DriverState};
+use crate::fail;
+use crate::logger;
+use crate::options::ChiselFlags;
+
+pub fn chisel_oneliner(flags: ChiselFlags) -> i32 {
+    let log_level = match flags.value_of("util.debugging") {
```

Comment:
This seems to be duplicated in `cmd_run`. Should fix this in a separate PR.

Reply:
:+1: